moveit_ros/move_group/src/move_group.cpp

```diff
@@ -200,7 +200,6 @@ int main(int argc, char** argv)
 
   std::shared_ptr<tf2_ros::Buffer> tf_buffer =
       std::make_shared<tf2_ros::Buffer>(nh->get_clock(), tf2::durationFromSec(10.0));
-  std::shared_ptr<tf2_ros::TransformListener> tfl = std::make_shared<tf2_ros::TransformListener>(*tf_buffer);
```

Comment:
We shouldn't rely on the CurrentStateMonitor to fill the tf_buffer. The TransformListener is required for the PlanningSceneMonitor to work without it.

Reply:
I think this comment is obsolete now, since parallel functionality was implemented through subscribers on the `/tf` topic.

Reply:
How is this comment obsolete? The subscribers are initialized inside the CSM so unless we call `PSM::startStateMonitor()` there won't be any listeners updating the buffer. A reasonable workaround would be to move the transform subscribers into the PSM and to call `CSM::updateMultiDofJoints` from there. What do you think?

Reply:
We do always call [startStateMonitor](https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp#L108) when constructing move_group